projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -349,10 +351,11 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
         if (this._groupChanges$) {
             this._groupChanges$.unsubscribe();
         }
-
-        this._ngZone.runOutsideAngular(() => {
-            this._resizeObserver.disconnect();
-        });
+        if (this.platformUtil.isBrowser) {
```

Comment:
I think you should make a general check for `if (this._resizeObserver)` rather than specific for platform browser.